### PR TITLE
2258 display dynamic countries in the side bar

### PR DESCRIPTION
**Pull request by sergioloporto**

### CODE REVIEW

**Review by maciekz1996 — CHANGES_REQUESTED** [review]
Fetching total number of cases and displaying fetched countries looks and works well 👍 There are however few things that I noticed apart from the comments that I left:

- Total number of cases in Sidebar is hardcoded and should be dynamic
- The autocomplete feature doesn't work. After selecting a country still all other countries are visible
- After clicking on a country name map is not zooming to selected country 
- There are no loading indicators in the sidebar, so when the app is loading for the first time the Sidebar is mostly blank. It would be best to add some animated `Skeletons` from `mui` library
- I also noticed that in the current application there is a button at the bottom of the Sidebar. We should add the same button in this application (you can see the screenshot below)

Screen Shot 2021-12-08 at 13 54 27

File: src/models/VariantsData.ts [hunk 1]
```diff
@@ -1,91 +1,91 @@
 export interface VariantsDataRow extends Record<string, number | string> {
-    location_id: string;
```
[the remark this answers is not in the document]
Comment (sergioloporto) [reply src/models/VariantsData.ts:2]: To make it more human readable, it is now ordered alphabetically

File: cypress/plugins/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```
Comment (maciekz1996) [inline cypress/plugins/index.js:1]: I think this file can be deleted completely as we are not using it

File: src/components/SideBar/index.tsx [hunk 3]
```diff
@@ -159,6 +196,14 @@ const SideBar = () => {
             <div id="sidebar-tab" onClick={handleOnClick}>
                 <span id="sidebar-tab-icon">{openSidebar ? '◀' : '▶'}</span>
             </div>
+            <GhListButtonBar
+                id="ghlist"
+                as="a"
+                href="https://data.covid-19.global.health/"
```
Comment (maciekz1996) [inline src/components/SideBar/index.tsx:202]: This url should be replaced with `.env` variable in order to use appropriate links in Dev and Prod environments

**Review by maciekz1996 — APPROVED** [review]
Great job, the code looks good and the functionality works as expected 🥇 There are two minor comments that I left, but other than that it can be merged. One missing thing though is Cypress tests, maybe it would be a good idea to create a new task for that.

**Review by maciekz1996 — CHANGES_REQUESTED** [review]
Oh actually I've just noticed that there is on functionality missing: 
After a user goes to other views like "Regional view" or "Coverage view" then "Fly to country" functionality is not working. All you have to do is to duplicate this functionality across all views 👍

**Review by maciekz1996 — APPROVED** [review]

**Pull request merged**